app/components/key_pair_component.js

```diff
@@ -0,0 +1,40 @@
+import { trackEvent } from '@18f/identity-analytics';
+
+class KeyPairGenerator extends HTMLElement {
```

Comment:
We call this by three different names. Can we pick one?

- "kp" (`<lg-kp />`)
- "key pair generator" (`KeyPairGenerator`)
- "key pair" (`KeyPairComponent`)

Reply:
Yes, but to make sure I'm clear, we want the following?

- `<lg-key-pair-generator>`
- `class KeyPairGeneratorElement extends HTMLElement`
- `class KeyPairGeneratorComponent < BaseComponent`

and move the component definition to files named `app/components/key_pair_generation_component.*`

Correct?

Reply:
Done in 33de67c92

Reply:
> Yes, but to make sure I'm clear, we want the following?
>
> Done in [8620fb1](https://github.com/18F/identity-idp/commit/8620fb165a415323c370c4ea0177eabdd4165e7d)

Yep, looks good 👍